Breaking Changes.md

```diff
@@ -6,3 +6,4 @@
 * Changed `modifyBabelrc` to `onCreateBabelConfig`
 * Changed `modifyWebpackConfig` to `onCreateWebpackConfig`
 * inlining css changed — remove it from any custom html.js as done automatically by core now.
+* manually install `react` and `react-dom`
```

Comment:
bunch of plugins moved dependencies to peerDependencies as well (`gatsby-plugin-glamor` and  `gatsby-plugin-typography` for example)

Reply:
Good point

Reply:
updated!